homeassistant/components/climate/fritzbox.py

```diff
@@ -113,6 +115,8 @@ def set_temperature(self, **kwargs):
     @property
     def current_operation(self):
         """Return the current operation mode."""
+        if self._device.holiday_active:
+            return STATE_AWAY
```

Comment:
You are not allowed to make up states. Only states from as defined by the climate component are allowed. For away mode, set the away mode toggle.

Reply:
That's something you have to handle yourself inside your entity.